modules/org.restlet.ext.jaxrs/src/main/java/org/restlet/ext/jaxrs/internal/resteasy/RestletHttpResponse.java

```diff
@@ -128,6 +128,11 @@ public void reset() {
 
     }
 
+    @java.lang.Override
+    public void flushBuffer() throws IOException {
```

Comment:
I'm not fond of empty methods, I try to avoid them if possible:
* if they are not called in real-life => throw with an explicit message. When it breaks, the error will be explicit
* if the method SHOULD do nothing, add a comment to explain why and a `debug` log to make sure someone debugging knows that a call to `flushBuffer` on this concrete type was called and did nothing.

Reply:
I guess you don't had a look at the whole class...

Reply:
I don't understand this class, it's in an internal package, and not used in the extension...